packages/video_player/video_player_avfoundation/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: video_player_avfoundation
 description: iOS and macOS implementation of the video_player plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/video_player/video_player_avfoundation
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
-version: 2.5.7
+version: 2.6.0
```

Comment:
[Comment from previous PR](https://github.com/flutter/packages/pull/6634#discussion_r1589477507) by @stuartmorgan:
Optional: I would probably make this a minor bump, since it's more of a new feature than a fix.